main/js/get_answer_GHJGDJHGJDHG.js

```diff
@@ -0,0 +1,37 @@
+function getkey(e){
+if(e.keyCode==13){
+$('.uibutton').show(300);
+    get_answer();
+}
+}
+
+function hide_answer(){
+    var answer_box = document.getElementById("answer");
+    //answer_box.hide();
+    answer_box.innerHTML = "<p>Hey, ask me a question!</p>";
+}
+
+
+function get_answer(){
+    question_box = document.getElementById("question_box");
+    question = encodeURIComponent(question_box.value);
+    //question = question_box.value;
+    
+$.ajax({type:'GET',url:'http://10.4.3.65/web2py/ChatBot_V4/default/get_answer',success:function(temp){ 
```

Comment:
That IP appears in 3 places. You guys decide what to do with it. I'll leave it for now.